code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm

```diff
@@ -353,7 +355,9 @@
 /datum/action/xeno_action/set_xeno_lead/proc/unset_xeno_leader(mob/living/carbon/xenomorph/selected_xeno, feedback = TRUE)
 	var/mob/living/carbon/xenomorph/xeno_ruler = owner
 	if(feedback)
+		xeno_ruler.balloon_alert(xeno_ruler, "Xeno demoted")
 		to_chat(xeno_ruler, span_xenonotice("We've demoted [selected_xeno] from Lead."))
+		selected_xeno.balloon_alert(selected_xeno, "Leadership removed")
 		to_chat(selected_xeno, span_xenoannounce("[xeno_ruler] has demoted us from Hive Leader. Our leadership rights and abilities have waned."))
```

Comment:
remove to chat?

Reply:
Retaining the to_chats here is from personal experience.

The xeno leadership selection menu is extremely poor (on my to-do list to improve). I often accidentally select already promoted xenos for leadership, thus demoting them.

Removing the name from the demotion message would mean spending just a little more time trying to figure out who I accidentally demoted and then re-promote them before finding another xeno to promote.

Hrm, I'll get onto fixing that immediately with at least a bandaid solution. Pie in the sky is of course a TGUI hive status.

Reply:
Resolved in new PR.